src/appkit.cs

```diff
@@ -7457,7 +7457,7 @@ interface NSGestureRecognizer : NSCoding {
 		Selector Action { get; set; }
 
 		[Export ("state")]
-		NSGestureRecognizerState State { get; }
+		NSGestureRecognizerState State { get; [Advice ("For Subclass Use.")] set; }
```

Comment:
I'm not a huge fan of the wording of the advice, I find it a little unclear and I had to open the bug to be sure.

I'd go for something more verbose: `Only subclasses of 'NSGestureRecognizer' can set this property.`

Also there's no reason to have capital letter for "Subclass" and "Use" (at least let's fix that).